sdk/eventhubs/ci.yml

```diff
@@ -35,10 +35,4 @@ stages:
       ServiceDirectory: eventhubs
       Artifacts:
         - name: azure-eventhubs
```

Comment:
Should this be: azure-messaging-eventhubs

Reply:
Yes - fixed in next push.